dist/sizzle.js

```diff
@@ -150,6 +150,24 @@ var i,
 				String.fromCharCode( high >> 10 | 0xD800, high & 0x3FF | 0xDC00 );
 	},
 
+	// CSS string/identifier serialization
+	// https://drafts.csswg.org/cssom/#common-serializing-idioms
+	rcssescape = /([\0-\x1f\x7f]|^-?\d)|^-$|[^\x80-\uFFFF\w-]/g,
+	fcssescape = function( ch, numeric ) {
+
+		// We allow NULL input, unlike the draft spec as of 2015-11-09
+		// Our syntax is already broader elsewhere
+		//if ( ch === "\0" ) {
+		//	throw new Error( "Cannot escape U+0000: " + str );
+		//}
+		if ( numeric ) {
+			// Code-point-escape the offending character, preserving any non-escaped prefix
+			return ch.slice( 0, -1 ) + "\\" + ch.charCodeAt( ch.length - 1 ).toString( 16 ) + " ";
```

Comment:
It is necessary to set the escape sequence apart from following hexadecimal characters (e.g., so "a.b" gets escaped as `a\2e b` instead of `a\2eb`, which corresponds to "a˫"). See https://drafts.csswg.org/cssom/#escape-a-character-as-code-point .